src/main/java/com/mastercard/developer/oauth/OAuth.java

```diff
@@ -174,7 +174,10 @@ static String toOauthParamString(SortedMap<String, List<String>> queryParamsMap,
    * @return UUID with dashes removed
```

Comment:
This comment has to be updated as it's not a "UUID with dashes removed" we return anymore. 

UUID: 1c9add51-4fd2-4f38-8e67-a6d22c054f03
Before: 1c9add514fd24f388e67a6d22c054f03
Now: fnr1q6smq30rnwu5